english/BayesTheorem/puppies/bayes-and-puppies.md

```diff
@@ -0,0 +1 @@
+When learning about [Bayes Theorem](https://en.wikipedia.org/wiki/Bayes'_theorem), why not learn with [puppies](https://sites.google.com/site/doingbayesiandataanalysis/)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.